implementations/rust/ockam/ockam_api/src/lib.rs

```diff
@@ -171,6 +171,29 @@ impl DefaultAddress {
     pub const KAFKA_CONSUMER: &'static str = "kafka_consumer";
     pub const KAFKA_PRODUCER: &'static str = "kafka_producer";
     pub const RPC_PROXY: &'static str = "rpc_proxy_service";
+
+    pub fn is_valid(name: &str) -> bool {
+        matches!(
```

Comment:
I learned a new macro today :-)

Reply:
I added a test for this function to verify that the macro works as expected in this case